README.md

```diff
@@ -70,6 +70,11 @@ The default is `.` ( root of the repository).
 Optional. The tflint version to install and use.
 The default is `latest`.
 
+### `tflint_rulesets`
+
+Optional. Space separated, official (from the terraform-linters GitHub organization) tflint rulesets to install and use.
+Default is `` (empty).
```

Comment:
How about mentioning `TFLINT_PLUGIN_DIR` to let users know how to use other custom rulesets by themselves.

Reply:
Good idea! Added!